rebasehelper/application.py

```diff
@@ -708,7 +717,7 @@ def run(self):
                     Application.check_build_requires(self.spec_file)
                 # Build packages
                 try:
-                    build = self.build_packages()
+                    build = self.build_packages
```

Comment:
Well this one is really strange. In my code and in my local commits I have it without property? Does anybody know how the hell the property came from?